### PR TITLE
Add Cover platform to Insteon component

### PR DESCRIPTION
**Pull request by teharris1**
**Description:**
Add Instoen cover entities.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#16215

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -20,3 +20,4 @@ source/.jekyll-metadata
 /.vs/slnx.sqlite-journal
 /.vs/VSWorkspaceState.json
 .vscode
+/.vs/home-assistant.io/v15/.suo
```
Comment (MartinHjelmare) [inline .gitignore]: Is this intended?

File: source/_components/cover.insteon.markdown [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+description: "Instructions on how to setup the Insteon cover locally within Home Assistant."
+date: 2018-08-20 16:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: insteon.png
+ha_category: Cover
+ha_iot_class: "Local Push"
+ha_version: 0.77
```
Comment (MartinHjelmare) [inline source/_components/cover.insteon.markdown]: 0.78

**Comment by teharris1** [comment]
Updated per request

**Review by MartinHjelmare — APPROVED** [review]
Looks good!

**Pull request merged**